bzt/cli.py

```diff
@@ -179,8 +187,6 @@ def perform(self, configs):
         else:
             self.log.info("Done performing with code: %s", self.exit_code)
 
-        self.__close_log()
```

Comment:
This used to be the very last thing because of Windows. Did you make sure that it will do fine on Windows with closing fd of temp file and moving file?